layouts/trackers/single.html

```diff
@@ -93,6 +95,10 @@ <h2>{{ T "tracker-single-transmitted-data-heading" }}</h2>
                                     -->
                                     {{ if or (strings.HasPrefix .reasoning "https://") (strings.HasPrefix .reasoning "http://") }}
                                         <a href="{{ .reasoning }}"><img src="/svg/information.svg" class="inline-icon icon-information" alt="{{ T "tracker-single-transmitted-data-table-reasoning-link-desc" }}" title="{{ T "tracker-single-transmitted-data-table-reasoning-link-desc" }}"></a>
+                                        {{ if isset $urlToArchivedUrl .reasoning }}
+                                            {{ $archivedUrl := index (index $urlToArchivedUrl .reasoning) (printf "%s/%s#%s" $.Params.tracker.slug $.Params.slug $property) }}
+                                            <a href="{{ $archivedUrl.archivedUrl }}"><img src="/svg/archive.svg" class="inline-icon icon-archive" alt="{{ T "tracker-single-transmitted-data-table-reasoning-link-archive" }}" title="{{ T "tracker-single-transmitted-data-table-reasoning-link-archive" }}"></a>
+                                        {{ end }}
```

Comment:
I am not so sure about that. Do we really want to display this icon for every link? That looks a bit silly, doesn't it? And in the majority of cases, people won't actually need the archived link…

Reply:
Maybe we could instead have a "References" section at the end of the page (maybe even in a `<details>` tag) that lists the link and archived link for each path?

Not quite sure what the right format for that would be, but I could imagine the general idea working quite well.